### ISSUE TITLE
pep8 segmentation fault

### ISSUE THREAD

**Issue by Nachtfeuer** [squeaky-pl/portable-pypy#51]
Running **pypy for Python 3.5** compatible I'm running into an issue with pep8:
It's to mention that PYTHON_VERSION for py27, py33, py34, py35, py36 and pypy is working fine;
of course pypy representing the Python 2.7 compatible version (see script).

```bash
# assume you have Docker installed
git clone https://github.com/Nachtfeuer/concept-py.git
cd concept-py
PYTHON_VERSION=pypy3 scripts/run_python.sh
```

You can verify the installation instructions in the switch ... case for **pypy3**.
The pep8 tool is basically running; when I do it manually with the defaults I see the line length warnings but the tool finally crashes with a segmentation fault. I tried different versions of pep8 without success.

**Comment by squeaky-pl** [comment]
It looks like it was fixed in PyPy3 5.8. If I subsitute 5.7.1 with 5.8 this doesnt happen anymore.

**Comment by Nachtfeuer** [comment]
Definitely. It does work all fine now :)
Thanks
